src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.spec.ts

```diff
@@ -120,4 +120,228 @@ describe('MissionRuntimeStepComponent', () => {
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let missionGroupItem = missionsSection.querySelector('.list-group-item');
+    let radioBtn = <HTMLInputElement>missionGroupItem.querySelector('input[type="radio"]');
+    let beforeChange = missionGroupItem.classList.contains('selected-list-item');
+    radioBtn.click();
+    tick();
+    fixture.detectChanges();
+    expect(beforeChange === false && missionGroupItem.classList.contains('selected-list-item') === true).toBeTruthy();
+  }));
+
+  it('should update the selected mission in launcher component', fakeAsync(() => {
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let missionGroupItem = missionsSection.querySelector('.list-group-item');
+    let radioBtn = <HTMLInputElement>missionGroupItem.querySelector('input[type="radio"]');
+    let beforeChange = missionGroupItem.classList.contains('selected-list-item');
+    radioBtn.click();
+    tick();
+    fixture.detectChanges();
+    expect(component.launcherComponent.summary.mission).toBe(component.missions[0]);
+  }));
+
+  it('should have the suggested missions tag when mission.suggested field is true', () => {
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let featuredTag = missionsSection.querySelector('.f8launcher-tag-featured');
+    expect(featuredTag).toBeDefined();
+  });
+
+  it('should not have the suggested missions tag when mission.suggested field is false', () => {
+    component.missions[0].suggested = false;
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let featuredTag = missionsSection.querySelector('.f8launcher-tag-featured');
+    expect(featuredTag).toBeNull();
+  });
+
+  it('should have the mission name as specified', () => {
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let missionHead = missionsSection.querySelectorAll('.list-group-item-heading');
+    let missions = component.missions, len = missions.length;
+    for (let i = 0; i < len; ++ i) {
+      expect((<HTMLDivElement>missionHead[i]).innerText).toBe(missions[i].name);
+    }
+  });
+
+  it('should have the mission description as specified', () => {
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let missionHead = missionsSection.querySelectorAll('.list-group-item-text');
+    let missions = component.missions, len = missions.length;
+    for (let i = 0; i < len; ++ i) {
+      expect((<HTMLDivElement>missionHead[i]).innerText).toBe(missions[i].description);
+    }
+  });
+
+  it('should show more if the url is present', () => {
```

Comment:
This test is failing because the UI currently does not show a 'more' link associated with this url. I'm thinking this test can be removed?